com.sap.cloud.lm.sl.cf.persistence/src/main/java/com/sap/cloud/lm/sl/cf/persistence/query/providers/SqlFileQueryProvider.java

```diff
@@ -111,60 +112,15 @@ public SqlFileQueryProvider(String tableName, DataSourceDialect dataSourceDialec
         };
     }
 
-    public SqlQuery<Integer> getDeleteAllQuery(final String space, final String namespace) {
-        return (Connection connection) -> {
-            PreparedStatement statement = null;
-            try {
-                statement = connection.prepareStatement(getQuery(DELETE_FILES_BY_NAMESPACE));
-                statement.setString(1, namespace);
-                statement.setString(2, space);
-                int deletedFiles = statement.executeUpdate();
-                logger.debug(MessageFormat.format(Messages.DELETED_0_FILES_WITH_SPACE_1_AND_NAMESPACE_2, deletedFiles, space, namespace));
-                return deletedFiles;
-            } finally {
-                JdbcUtil.closeQuietly(statement);
-            }
-        };
-    }
-
-    public SqlQuery<Integer> getDeleteModifiedBeforeQuery(Date modificationTime) {
-        return (Connection connection) -> {
-            PreparedStatement statement = null;
-            try {
-                statement = connection.prepareStatement(getQuery(DELETE_FILES_MODIFIED_BEFORE));
-                statement.setTimestamp(1, new java.sql.Timestamp(modificationTime.getTime()));
-                int deletedFiles = statement.executeUpdate();
-                logger.debug(MessageFormat.format(Messages.DELETED_0_FILES_MODIFIED_BEFORE_1, deletedFiles, modificationTime));
-                return deletedFiles;
-            } finally {
-                JdbcUtil.closeQuietly(statement);
-            }
-        };
-    }
-
-    public SqlQuery<Boolean> getDeleteFileAttributesQuery(final String space, final String id) {
-        return (Connection connection) -> {
-            PreparedStatement statement = null;
-            try {
-                statement = connection.prepareStatement(getQuery(DELETE_FILE_BY_ID));
-                statement.setString(1, id);
-                statement.setString(2, space);
-                int deletedRows = statement.executeUpdate();
-                logger.debug(MessageFormat.format(Messages.DELETED_0_FILES_WITH_ID_1_AND_SPACE_2, deletedRows, id, space));
-                return deletedRows > 0;
-            } finally {
-                JdbcUtil.closeQuietly(statement);
-            }
-        };
-    }
-
     public SqlQuery<List<FileEntry>> getListFilesQuery(final String space, final String namespace) {
         return (Connection connection) -> {
             PreparedStatement statement = null;
             ResultSet resultSet = null;
             try {
                 List<FileEntry> files = new ArrayList<>();
-                if (space == null) {
+                if (space == null & namespace == null) {
+                    statement = connection.prepareStatement(getQuery(SELECT_ALL_FILES));
```

Comment:
add a separate getListAllFilesQuery since it seems to be part of the usual process

Reply:
Done.